lib/banyan_api/app_install.ex

```diff
@@ -64,15 +64,13 @@ defmodule BanyanAPI.AppInstall do
   end
 
   @spec format_shop_params(map()) :: map()
+  # WARNING the only supported keys right now are settings, this func needs to handle both strings and atoms as keys in the map.
```

Comment:
👍 🙇

Reply:
:D I figure future us will appreciate that